### DISCUSSION TITLE
On-call schedule how-tos?

### DISCUSSION THREAD

**Discussion by caioaao** [Netflix/dispatch#2577] [Q&A]
How would one go to set up an on-call schedule so that only on-call folks are assigned to an incident?

**Comment by mvilanova** [comment]
We have a PagerDuty oncall team with an oncall service that is linked to an escalation policy which uses an escalation schedule containing the people that will be oncall. Then we have an oncall service set up in Dispatch that's the target service in the Incident Commander role policy.
